### ISSUE TITLE
Mod's balance suggestion

### ISSUE THREAD

**Issue by Aizekpanin94** [RayzoRayy/RaysDongles#13]
So, i've managed to play your mod. And i should say that amount of work is incredible, it's one hell of a mod. But RayDongles has a big problem. Balance problem. 

I figured it out in the first 5 minutes of playing. I spawned right next to a plain village where i found two emeralds. I immediately crafted a sword just to be... Upset. Emerald sword has 9 attask damage, when diamond sword, best vanilla sword, has 7 attack damage. Something's off, right? Whole idea of seeking for diamonds and new rare ores had been crushed by a pair os emeralds in the village. I played for another couple of hours and i found out more disbalanced ores. Then i decided to prove my theory.

So i got to another world and created this 
![Безымянный](https://user-images.githubusercontent.com/54679259/63959396-bd374f80-ca94-11e9-81bd-c1a22b2a1ae3.png)
A whole set of materials that could be used in gear making, except wood. No one uses wood. 
I had some experiments with gear and armor and i made a conclusion: **a whole mod needs to be balanced.**
First of all, Emeralds, Those are kinda common in the Otherworld, you can always find a couple or more in villages. Problem is, emerald gear is equal, if not stronger than diamond gear, best gear in vanilla. So instead of obtaining ores in caves and having fun you can just go travel, find nearest village and rob it, giving yourself 2-6 emeralds. 
Second thing is Ruby. Ruby is common in caves, like really common. I can just dig up a hole and find about half a stack of them. And what's worse, you can obtain ruby with a stone pickaxe. **Stone pickaxe**. And when you find ruby, you can make yourself an armor set equal to diamond set, gear slightly stronger than iron gear and a sword equal to emerald sword. It's a shame. 
In addition to all of this, i found out that pretty much every sword (expept salad-green ore between ruby and violet ore, it has 5 damage) has 9 damage. Pickaxes, Axes and Shovels have a big gap between their theoretical rarity, practical rarity and power. Like you can obtain ore that can be obtained only by diamond+ pickaxe and gear from this ore will be far worse than diamond+. 

My next point of interest was top-notch raid gear like withered and topaz. There is another problem. Topaz is also common and it can be obtained with emerald pickaxe. All that you need to make top-notch armor is to throw it in furnace to make a hardened topaz. After that you will have armor stronger than vanilla diamond armor. If topaz was more rare than diamond, i wouldn't talked about it, but diamond is still rarest ore in game. I've managed to found about 20 units of topaz in one go and about 4 units of diamonds. 

After that i've tested rarity of topaz gear and withered gear. For that experiment i had to kill Wither.
![Безымянный 2](https://user-images.githubusercontent.com/54679259/63961914-82d0b100-ca9a-11e9-88b9-bacfccacc9bf.png)
Result was interesting. Instead of raiding wither one time, obtaining nether star, making topaz gear and then raiding wither another couple of times you can just raid wither one single time and obtain best gear in the game. I don't talk about withered set, it needs 19 nether star chunks.
From one wither raid you can do one Topaz Sword and one Topaz Paxel, or you can do one Withered Sword, one Withered pickaxe, one withered shovel and one withered axe. I think that every player will choose withered gear.
All of it goes from quantity of chunks in one nether star - 9 chunks.
![Безымянный 3](https://user-images.githubusercontent.com/54679259/63963240-45215780-ca9d-11e9-9df5-f21498bd14d1.png)

And now suggestions. I might suggest to balance gear - nerf emerald gear, make it somewhat equal to iron armor, like sapphire gear. Work on ores and make some of them on your choice unpickable to diamond pickaxe. Make player evolve to better gear. 
Work on rarity of resourses and think of raid sets like topaz gear and withered gear. Not every player would want to go for all withered gear because he will stand between two options - withered pickaxe, shovel, axe and sword or topaz paxel and topaz sword. 

Thank you for reading this, i'm counting on your response.

**Comment by Aizekpanin94** [comment]
TL;DR: Mod doesn't have ladder-like progress of ores, some of ores are imbalanced, there is strange choice of raid gear after raid boss. suggestion was given.

**Comment by RayzoRayy** [comment]
Hi. Thanks for the suggestions. This is something ive wanted to work on for
a awhile now, it has just been rough sitting down trying to figure out what
to do with the balancing stuff. With a full time job now, i havent had much
time latley to sit down and figure it all out. I shall take your suggestion
though and try to work on more balancing in the future. I appreciate the
truly detailed write up of your post. Thank you.

**Comment by RayzoRayy** [comment]
Just to keep you informed, I didn't forget about this issue. I have released an update just now that brings SOME balance changes. More will come over time, this is just the start of this very slow process, as well as some other bug fixes. Going foward, this update will only be for 1.15.1 as right now this is my primary focus for modding, as it seems this is the direction other modders, and Forge is now moving towards. While this will take time, I simply do not have the time to maintain multiple versions of minecraft, when eventually the "older" versions will become obsolete. Thanks again for this amazing write-up of your suggestions.

**Comment by RayzoRayy** [comment]
This suggestion is pretty old, but regardless leaving this comment for anyone that may stumble across it

Tool Materials have been re-balanced in my 1.18.2, 1.19.2, and 1.19.3 versions 4.14, 5.14, and 6.14 respectively. My modded materials are now more in line with Vanilla without being super OP. Armor Materials still need to be re-balanced, but ALL tools in the versions listed above have been re-balanced.
These changes will NOT be back ported to versions under 1.18.2. Armor Material rebalance will come over the next few updates.
